docs/source/faq/index.rst

```diff
@@ -0,0 +1,45 @@
+FAQ
+===
+
+I'm new. Where to start?
+------------------------
+
+Hello, newcomer! You are very welcome :) \
+There are 2 ways for you to start with Iroha: \
+1. You can see what Iroha is an how it works by building a simple example network following our `Getting Started Guide <https://iroha.readthedocs.io/en/latest/getting_started/index.html>`_ \
+2. You can get acquainted with the `core concepts of Iroha <https://iroha.readthedocs.io/en/latest/core_concepts/index.html>`_ and start building your own `Iroha network <https://iroha.readthedocs.io/en/latest/guides/index.html>`_ \\
```

Comment:
Double backslash is rendered as single backslash in html.
Please remove all backslashes from the end of lines.

![image](https://user-images.githubusercontent.com/36847043/47715490-fd4d5c00-dc50-11e8-8203-ea4a895e393f.png)